src/mono/wasm/test-main.js

```diff
@@ -101,18 +101,12 @@ if (typeof globalThis.performance === 'undefined') {
         }
     }
 }
-
 var Module = {
-    no_global_exports: true,
-    mainScriptUrlOrBlob: "dotnet.js",
     config: null,
     configSrc: "./mono-config.json",
-    print: console.log,
-    printErr: console.error,
-    onConfigLoaded: function () {
+    onConfigLoaded: () => {
         if (!Module.config) {
-            console.error("Could not find ./mono-config.json. Cancelling run");
-            fail_exec(1);
+            set_exit_code(1, "Could not find ./mono-config.json. Cancelling run");
```

Comment:
while i think the extra code below running is harmless, we should probably 'return' here after setting the exit code anyway